trustfall_core/src/interpreter/execution.rs

```diff
@@ -241,6 +241,24 @@ fn construct_outputs<'query, AdapterT: Adapter<'query>>(
     }))
 }
 
+/// takes a FieldValue and produces a usize clamped to 0 and usize::MAX
```

Comment:
```suggestion
/// Takes a FieldValue and produces a usize clamped between 0 and `usize::MAX`.
```
Docstrings should always be capitalized and use code blocks for code.